### DISCUSSION TITLE
✅ More Robust Slides Widget with Dynamic Content

### DISCUSSION THREAD

**Discussion by cranstantinople** [elementor/elementor#17346] [Feature Request]
Would be awesome if the slides widget was a little more robust and could use dynamic content with queries for custom post types.

There are several sites that I have to use a completely separate slider plugin because the slides widget is missing these basic features.  A couple of examples where I'd like to use this.

Store header slides to query items by sale or category.

Rotating slides of upcoming events.

Rotating slides of recent posts.

**Comment by nicholaszein** [comment]
Hello there @cranstantinople!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.11.0! 🥳 

You can now use the Loop Carousel widget and build your own designs! Check the [documentation in the Help Center](https://elementor.com/help/loop-carousel/)!

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
